src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -848,13 +848,30 @@ typedef VOID (*RtcOnFrame)(UINT64, PFrame);
 
 /**
  * @brief RtcOnBandwidthEstimation is fired everytime a bandwidth estimation value
- * is computed. This will be fired for sender or receiver side estimation
+ * is computed. This will be fired for receiver side estimation
  *
  * NOTE: RtcOnBandwidthEstimation is a KVS specific method
  *
  */
 typedef VOID (*RtcOnBandwidthEstimation)(UINT64, DOUBLE);
 
+/**
+ * @brief RtcOnSenderBandwidthEstimation is fired everytime a bandwidth estimation value
+ * is computed by sender. This is an estimate of ALL packets sent across all transceivers.
+ * See https://tools.ietf.org/html/draft-holmer-rmcat-transport-wide-cc-extensions-01 for more details.
+ *
+ * NOTE: RtcOnSenderBandwidthEstimation is a KVS specific method
+ *
+ * @param[in] UINT64 User customData that will be passed along when RtcOnSenderBandwidthEstimation is called
+ * @param[in] UINT32 txBytes - bytes sent over the transport
+ * @param[in] UINT32 rxBytes - bytes reported as received
+ * @param[in] UINT32 txPackets - number of packets sent over the transport
+ * @param[in] UINT32 rxPackets - number of packets reported as received
+ * @param[in] UINT64 duration - time window for txBytes, rxBytes, txPackets, rxPackets
+ *
+ */
+typedef VOID (*RtcOnSenderBandwidthEstimation)(UINT64, UINT32 txBytes, UINT32 rxBytes, UINT32 txPackets, UINT32 rxPackets, UINT64 duration);
```

Comment:
Nit for consistency:
`typedef VOID (*RtcOnSenderBandwidthEstimation)(UINT64, UINT32 txBytes, UINT32 rxBytes, UINT32 txPackets, UINT32 rxPackets, UINT64 duration);` to 

`typedef VOID (*RtcOnSenderBandwidthEstimation)(UINT64, UINT32, UINT32, UINT32, UINT32, UINT64);`